src/PrestaShopBundle/Service/Grid/ResponseBuilder.php

```diff
@@ -133,7 +134,7 @@ public function buildSearchResponse(
 
         $redirectUrl = $this->router->generate($redirectRoute, $redirectParams);
 
-        return new RedirectResponse($redirectUrl, 302);
+        return new RedirectResponse($redirectUrl, Response::HTTP_FOUND);
```

Comment:
This is default, you can remove the 2nd parameter.

Reply:
That makes sense, no opinion on this one the const is already better anyway so either of the choices is fine for me I'm just worries if Symfony could someday change the default value, but I guess it would be a breaking change they don't have much interest to do so it should be safe to remove the parameter